lib/hanami/events/formatter/plain_text.rb

```diff
@@ -0,0 +1,21 @@
+module Hanami
+  module Events
+    class Formatter
+      class PlainText
+        def initialize(data)
+          @data = data
+        end
+
+        def format
+          "Events:\n#{formatted_events}"
+        end
+
+        private
+
+        def formatted_events
+          @data.map { |d| "\t#{d[:title].inspect}" }.join("\n")
```

Comment:
To make sure that we're on the same page. I mean this could be renamed from `title` to `event_name`: 
https://github.com/hanami/events/blob/master/lib/hanami/events/subscriber.rb#L15
Because now we have 3 "names" for the same thing: `title`, `pattern`, `event_name`.
My suggestion is to be consistent and use `event_name`.